documentation/modules/security/con-certificate-authorities.adoc

```diff
@@ -26,3 +26,15 @@ You can configure the validity period of:
 
 * Cluster CA certificates in `Kafka.spec.clusterCa.validityDays`
 * Client CA certificates in `Kafka.spec.clientsCa.validityDays`
+
+== Options for installing your own CA certificates
+
+In your security model, client applications might need to trust certificates that were issued by an external CA, such as your company's internal CA or a public CA, in order to connect to external systems. {ProductName} provides two methods for configuring your Kafka client applications to connect to the brokers in your Kafka cluster using your own certificates.
+
+You can install your own certificates at two levels:
+
+* At a cluster level--see xref:installing-your-own-ca-certificates-{context}[].
+
+* For each listener (TLS and external) in a Kafka broker--see xref:proc-installing-certs-per-listener-{context}[].
+
+If you install your own certificates using one or both of these methods, they are not automatically renewed by the Cluster Operator. You must manually renew certificates according to your security model.
```

Comment:
Couple of mentions of _security model_. Would that be understood? How might it reflect on the manual renewal? Maybe we can drop and just say you need to manually renew the certs.

Reply:
I would probably just say _You must manually renew certificates when needed._ Which could be expiration or whatever security policy the user follows. I would be a bit worried that _security model_ might not have clear meaning and understanding for everyone.

Reply:
Removed "security model" and reworded as follows:

Your client applications might need to trust certificates that were issued by an external CA, such as your company's internal CA or a public CA, in order to connect to external systems.
To support this scenario, {ProductName} provides two methods for configuring your Kafka client applications to connect to the brokers in your Kafka cluster using your own certificates.

You can install your own certificates at two levels:

* At the cluster level--see xref:installing-your-own-ca-certificates-{context}[].

* For each listener (TLS or external listener, or both) in a Kafka broker--see xref:proc-installing-certs-per-listener-{context}[].

If you install your own certificates, they are not automatically renewed by the Cluster Operator.
In this case, you must manually renew certificates when needed.

Reply:
This was refactored again in response to Jakub's comment that the above module "mixes together" the two types of certificates.